testing.rst

```diff
@@ -484,7 +484,7 @@ script::
 AJAX Requests
 ~~~~~~~~~~~~~
 
-The Client provides a :method:`Symfony\\Component\\BrowserKit\\Client::xmlHttpRequest`
+The Client provides a :class:`Symfony\\Bundle\\FrameworkBundle\\KernelBrowser :: xmlHttpRequest` 
```

Comment:
This is no longer valid as there is no `xmlHttpRequest` method in `KernelBrowser `. This means that the code below is also incorrect and needs further changes.